### PR TITLE
Peppy.Osu! 2021.707.0

### PR DESCRIPTION
**Pull request by aeongdesu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20193)

### CODE REVIEW

File: manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.707.0-lazer
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.installer.yaml:5]:
```suggestion
PackageVersion: 2021.707.0
```

File: manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.707.0-lazer
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2021.707.0
```

File: manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.707.0-lazer
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.yaml:5]:
```suggestion
PackageVersion: 2021.707.0
```

File: manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+PackageLocale: en-US
+Publisher: ppy Pty Ltd
+PublisherUrl: https://osu.ppy.sh/home
+PublisherSupportUrl: https://github.com/ppy/osu/issues
+PrivacyUrl: https://osu.ppy.sh/legal/en/Privacy
+Author: Dean "peppy" Herbert
+PackageName: osu!lazer
```
Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.707.0-lazer/Peppy.Osu!.locale.en-US.yaml:12]:
```suggestion
PackageName: osu!
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Guess they changed the PackageName very recently 👀

Edit: Holy shit they changed it in this version, giga coincidence that i wanted to check if i could find the InstallerType lmao
image

**Comment by KeepSOBP** [comment]
Done :)

**Comment by KeepSOBP** [comment]
hey @OfficialEsco, what should I do?

**Comment by OfficialEsco** [comment]
> hey @OfficialEsco, what should I do?

Thought you were experienced enough, my bad, you need to rename the Version Folder
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\p\Peppy\Osu!\2021.707.0-lazer', 
expected 'manifests\p\Peppy\Osu!\2021.707.0'
```

**Comment by KeepSOBP** [comment]
Sorry, I didn't check error log ._.

**Comment by OfficialEsco** [comment]
> Sorry, I didn't check error log ._.

Naaah thats not the part i meant, i usually warn people that they need to change the folder/file names, but i didn't tell you that 🙄

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**